### ISSUE TITLE
Blogs do not support embedding Tweets

### ISSUE THREAD

**Issue by hamadax** [PHPfox-Official/phpfox-v4-issues#2649]
# Important

Please follow this template!

----------------------------------------

## What's happened?
Embedded tweets do not display properly.

## Steps to reproduce:
1. Start a blog
2. Embed a Tweet
3. View blog

## What's expected?
The blog to show embedded Tweets like this:

![image](https://user-images.githubusercontent.com/48427415/56968352-056b5300-6b31-11e9-92f5-d3fc67b6de6c.png)

## Browsers and Devices tested
(Example: Chrome on iPhone X, Safari on Macbook, Miscrosoft Edge on Windown 10, Firefox on Ubuntu 16.04, ...)

...All
## Server information
(Example: CentOS 7, php 7.1 apache)

...

## phpFox version
(Example: phpFox 4.6.0)

...

## Screenshots
...

**Comment by hamadax** [comment]
More info here: https://developer.twitter.com/en/docs/twitter-for-websites/embedded-tweets/overview

**Comment by harrison05** [comment]
Yes, we don't support embed an HTML block with CKEditor, because HTML tags will be escaping when saved.

Regards.
